src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -62,65 +102,65 @@ export const AddressModal: React.FC<Props> = ({
               type="text"
               title="Full Name"
               onChange={formik.handleChange}
-              error={formik.errors.name}
-              value={formik.values.name}
+              error={formik.errors?.name}
+              value={formik.values?.name}
```

Comment:
Ditto throughout PR. Happy to pair to get help understand / reduce the use of these!

Reply:
Here this is needed bc the component is now being repurposed in a way where the values can be null upon initial render (when creating a new address). Before these changes introduced by this PR, the values shouldn't be null because the form data was already populated (when editing an existing address).

Reply:
> Before these changes introduced by this PR, the values wouldn't be null because the form data was already populated (when editing an existing address)

There's still the shape of the data, however, which formik needs to initialize. 

Meaning: 

```js
const data = {
  name: null,
  location: null,
  age: null
}
```

which formik then uses to construct a formik bag which handles populating things like `errors` and `values`; one doesn't ever pass formik a totally empty object because then it wouldn't know what to init. And if `formik` is null from the get go it might be worthwhile to refactor the component so that its not. 

(I'm likely misunderstanding something but happy to look at it closer!)